<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera On</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_stop">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>110</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera Off</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Cam">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>601</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Detect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Fps On</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
